### PR TITLE
API-11290: document agent's `login-status`

### PR DESCRIPTION
**Pull request by litwinow**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira](https://livechatinc.atlassian.net/browse/API-11290)

# Description

Add missing documentation for agent's `login_status` field.

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

**Review by lpmichalik — APPROVED** [review]

File: src/pages/management/configuration-api/index.mdx [hunk 1]
```diff
@@ -344,22 +345,23 @@ Updates the properties of an Agent specified by `id`.
...
-| `work_scheduler.****.<work_scheduler>` | `object`   | No       | `<work_scheduler>` values __\*\*\*\*\*__                                           |
+| Parameter                              | Data type  | Required | Notes                                                                                                                             |
+| -------------------------------------- | ---------- | -------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| `id`                                   | `string`   | Yes      | Agent ID                                                                                                                          |
+| `name`                                 | `string`   | No       | Agent name                                                                                                                        |
+| `role`                                 | `string`   | No       | Agent role, should be one of the following: `viceowner`, `administrator`, `normal`                                                |
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/index.mdx:352]:
```suggestion
| `role`                                 | `string`   | No       | Agent role, should be one of the following: `viceowner`, `administrator`, `normal`.                                               |
```

File: src/pages/management/configuration-api/index.mdx [hunk 2]
```diff
@@ -344,22 +345,23 @@ Updates the properties of an Agent specified by `id`.
...
+| `avatar`                               | `string`   | No       | Avatar URL                                                                                                                        |
+| `login_status`                         | `string`   | No       | Agent's default status right after the login, should be one of the following: `accepting chats`, `not accepting chats` (default). |
+| `job_title`                            | `string`   | No       | Agent's job title                                                                                                                 |
+| `mobile`                               | `string`   | No       | Agent's mobile number                                                                                                             |
+| `max_chats_count`                      | `int`      | No       | Agent's maximum number of concurrent. Limited to 100. chats                                                                       |
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/index.mdx:357]:
```suggestion
| `max_chats_count`                      | `int`      | No       | Agent's maximum number of concurrent chats. Limited to 100.                                                                       |
```

File: src/pages/management/configuration-api/v3.2/index.mdx [hunk 3]
```diff
@@ -352,22 +353,23 @@ Updates the properties of an Agent specified by `id`.
...
-| `work_scheduler.****.<work_scheduler>` | `object`   | No       | `<work_scheduler>` values __\*\*\*\*\*__                                           |
+| Parameter                              | Data type  | Required | Notes                                                                                                                             |
+| -------------------------------------- | ---------- | -------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| `id`                                   | `string`   | Yes      | Agent ID                                                                                                                          |
+| `name`                                 | `string`   | No       | Agent name                                                                                                                        |
+| `role`                                 | `string`   | No       | Agent role, should be one of the following: `viceowner`, `administrator`, `normal`                                                |
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v3.2/index.mdx:360]:
```suggestion
| `role`                                 | `string`   | No       | Agent role, should be one of the following: `viceowner`, `administrator`, `normal`.                                               |
```

File: src/pages/management/configuration-api/v3.3/index.mdx [hunk 4]
```diff
@@ -346,22 +347,23 @@ Updates the properties of an Agent specified by `id`.
...
-| `work_scheduler.****.<work_scheduler>` | `object`   | No       | `<work_scheduler>` values __\*\*\*\*\*__                                           |
+| Parameter                              | Data type  | Required | Notes                                                                                                                             |
+| -------------------------------------- | ---------- | -------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| `id`                                   | `string`   | Yes      | Agent ID                                                                                                                          |
+| `name`                                 | `string`   | No       | Agent name                                                                                                                        |
+| `role`                                 | `string`   | No       | Agent role, should be one of the following: `viceowner`, `administrator`, `normal`                                                |
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v3.3/index.mdx:354]:
```suggestion
| `role`                                 | `string`   | No       | Agent role, should be one of the following: `viceowner`, `administrator`, `normal`.                                               |
```

File: src/pages/management/configuration-api/v3.5/index.mdx [hunk 5]
```diff
@@ -464,22 +465,23 @@ Updates the properties of an Agent specified by `id`.
...
-| `work_scheduler.****.<work_scheduler>` | `object`   | No       | `<work_scheduler>` values __\*\*\*\*\*__                                           |
+| Parameter                              | Data type  | Required | Notes                                                                                                                             |
+| -------------------------------------- | ---------- | -------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| `id`                                   | `string`   | Yes      | Agent ID                                                                                                                          |
+| `name`                                 | `string`   | No       | Agent name                                                                                                                        |
+| `role`                                 | `string`   | No       | Agent role, should be one of the following: `viceowner`, `administrator`, `normal`                                                |
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v3.5/index.mdx:472]:
```suggestion
| `role`                                 | `string`   | No       | Agent role, should be one of the following: `viceowner`, `administrator`, `normal`.                                               |
```

File: src/pages/management/configuration-api/v3.5/index.mdx [hunk 6]
```diff
@@ -464,22 +465,23 @@ Updates the properties of an Agent specified by `id`.
...
+| `role`                                 | `string`   | No       | Agent role, should be one of the following: `viceowner`, `administrator`, `normal`.                                               |
+| `avatar`                               | `string`   | No       | Avatar URL                                                                                                                        |
+| `login_status`                         | `string`   | No       | Agent's default status right after the login, should be one of the following: `accepting chats`, `not accepting chats` (default). |
+| `job_title`                            | `string`   | No       | Agent's job title                                                                                                                 |
+| `mobile`                               | `string`   | No       | Agent's mobile number                                                                                                             |
+| `max_chats_count`                      | `int`      | No       | Agent's maximum number of concurrent. Limited to 100. chats                                                                       |
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v3.5/index.mdx:477]:
```suggestion
| `max_chats_count`                      | `int`      | No       | Agent's maximum number of concurrent chats. Limited to 100.                                                                        |
```

Comment (kamilatokarska) [reply src/pages/management/configuration-api/v3.5/index.mdx:477]: I am sorry, I have not noticed this one during the first review

**Review by kamilatokarska — APPROVED** [review]
:)

**Pull request merged**